addons/website_event_meet/controllers/community.py

```diff
@@ -59,7 +61,7 @@ def _event_meeting_rooms_get_values(self, event, lang=None):
 
         return {
             # event information
-            "event": event.sudo(),
+            "event": event,
```

Comment:
Mmmh, do you know why there was a sudo, and why are you removing it ?

Reply:
Why sudo was there ? I am not sure either, seems its not required. Why is it removed ? Already explained in the last point of commit related to sponsors.

Reply:
Well if everything works, ok ^^